packages/liveblocks-node/src/authorize.ts

```diff
@@ -90,18 +91,18 @@ export async function authorize(
 
     if (!result.ok) {
       return {
-        status: 403,
+        status: 403 /* Unauthorized */,
         body: await result.text(),
       };
     }
 
     return {
-      status: 200,
+      status: 200 /* OK */,
       body: await result.text(),
     };
   } catch (er) {
     return {
-      status: 403,
+      status: 503 /* Service Unavailable */,
```

Comment:
The only real (and important) change here is this change of status code. That's a fixed bug. The rest is just cleanup while I was in this module.